### ISSUE TITLE
Next step after building this?

### ISSUE THREAD

**Issue by jwiedey** [Ultimaker/cura-build-environment#44]
Newbie question: I've built this project successfully. What's my next step?

The README says "This environment can be used in combination with the cura-build repository to produce Cura executables for all three supported platforms.". But cura-build seems to be set up to build the same Cura dependencies as cura-build-environment, rather than *use* the stuff built by cura-build-environment. Seems like I'm missing something really basic.

**Comment by Ghostkeeper** [comment]
> But cura-build seems to be set up to build the same Cura dependencies as cura-build-environment, rather than use the stuff built by cura-build-environment.

This is incorrect. Or at least not the way it's intended.

Cura-build-environment should build the dependencies of Cura that are not considered part of Cura. This includes a bunch of Python libraries like SciPy and PyQt, and some additional stuff that's required to package such as CX_Freeze.

Cura-build should build the repositories that we consider part of Cura itself. Mostly the stuff that is maintained by Ultimaker: Cura, Uranium, CuraEngine, etc. There is a bit of a vague border there though that is not really done consistently. For instance, cura-build will build libCharon while cura-build-environment builds libArcus, and both of these are maintained by Ultimaker and used in more projects than just Cura.

So in short, your answer: You're supposed to build the cura-build repository next.
